user_acceptance_tests/README.md

```diff
@@ -0,0 +1,51 @@
+# User Acceptance Tests
+
+## Purpose
+
+The purpose of these tests is to ensure that key deployment configurations remain deployable, through the changes we introduce into the repo. They mainly serve as safeguards against regressions, but new deployments & tests can be added as new features are added.
+
+## Definitions
+
+Before we go too far, some definitions for clarity:
+
+* `deployment configuration, deploy config` - a file containing the state of an environment that will be created (or torn down).
+
+* `scenario` - a deployment configuration meant to exercise some specific story or use case.
```

Comment:
I would avoid using the term `scenario` for this, the sets of actions that the simulator executes are called `scenarios`. Could we stick with using deployment configuration?

Reply:
i dont think this is too much of an issue, since someone who contributes / cares about the UATs is likely not a general user (someone who is using the simulator).

if we want a different word, i would not use deployment configuration since this captures extra context. off the top of my head, maybe story, case, use-case is something i would see as next in line.

additionally, the word scenario is also used in the workflow, would you like to see it changed there as well?

Reply:
No, that makes sense! Let's keep it `scenario`